posthog/hogql/database/schema/channel_type.py

```diff
@@ -37,61 +44,71 @@ def create_initial_channel_type(name: str):
         expr=parse_expr(
             """
 multiIf(
-    match(properties.$initial_utm_campaign, 'cross-network'),
+    match({campaign}, 'cross-network'),
     'Cross Network',
 
     (
-        match(properties.$initial_utm_medium, '^(.*cp.*|ppc|retargeting|paid.*)$') OR
-        properties.$initial_gclid IS NOT NULL OR
-        properties.$initial_gad_source IS NOT NULL
+        match({medium}, '^(.*cp.*|ppc|retargeting|paid.*)$') OR
+        {gclid} IS NOT NULL OR
+        {gad_source} IS NOT NULL
     ),
     coalesce(
-        hogql_lookupPaidSourceType(properties.$initial_utm_source),
-        hogql_lookupPaidDomainType(properties.$initial_referring_domain),
+        hogql_lookupPaidSourceType({source}),
+        hogql_lookupPaidDomainType({referring_domain}),
         if(
-            match(properties.$initial_utm_campaign, '^(.*(([^a-df-z]|^)shop|shopping).*)$'),
+            match({campaign}, '^(.*(([^a-df-z]|^)shop|shopping).*)$'),
             'Paid Shopping',
             NULL
         ),
-        hogql_lookupPaidMediumType(properties.$initial_utm_medium),
+        hogql_lookupPaidMediumType({medium}),
         multiIf (
-            properties.$initial_gad_source = '1',
+            {gad_source} = '1',
             'Paid Search',
 
-            match(properties.$initial_utm_campaign, '^(.*video.*)$'),
+            match({campaign}, '^(.*video.*)$'),
             'Paid Video',
 
             'Paid Other'
         )
     ),
 
     (
-        properties.$initial_referring_domain = '$direct'
-        AND (properties.$initial_utm_medium IS NULL OR properties.$initial_utm_medium = '')
-        AND (properties.$initial_utm_source IS NULL OR properties.$initial_utm_source IN ('', '(direct)', 'direct'))
+        {referring_domain} = '$direct'
+        AND ({medium} IS NULL OR {medium} = '')
+        AND ({source} IS NULL OR {source} IN ('', '(direct)', 'direct'))
     ),
     'Direct',
 
     coalesce(
-        hogql_lookupOrganicSourceType(properties.$initial_utm_source),
-        hogql_lookupOrganicDomainType(properties.$initial_referring_domain),
+        hogql_lookupOrganicSourceType({source}),
+        hogql_lookupOrganicDomainType({referring_domain}),
         if(
-            match(properties.$initial_utm_campaign, '^(.*(([^a-df-z]|^)shop|shopping).*)$'),
+            match({campaign}, '^(.*(([^a-df-z]|^)shop|shopping).*)$'),
             'Organic Shopping',
             NULL
         ),
-        hogql_lookupOrganicMediumType(properties.$initial_utm_medium),
+        hogql_lookupOrganicMediumType({medium}),
         multiIf(
-            match(properties.$initial_utm_campaign, '^(.*video.*)$'),
+            match({campaign}, '^(.*video.*)$'),
             'Organic Video',
 
-            match(properties.$initial_utm_medium, 'push$'),
+            match({medium}, 'push$'),
             'Push',
 
             'Other'
         )
     )
 )""",
             start=None,
+            placeholders={
```

Comment:
Moving these to placeholders will be useful in a later PR, where I use session properties instead of person properties